docs/ruleset_functions/is_zone_mechanically_cooled.md

```diff
@@ -0,0 +1,30 @@
+# is_zone_mechanically_cooled
+**Schema Version:** 0.0.22  
+
+**Description:** determines whether a zone is mechanically cooled or not.  DOES NOT CHECK FOR TRANSFER AIR
+
+**Inputs:**
+- **RMD**
+- **zone**
+
+**Returns:**  
+- **result**: a boolean - TRUE or FALSE
+ 
+**Function Call:**
+- **get_list_hvac_systems_associated_with_zone**
+
+## Logic:
+- set result to FALSE: `result = FALSE`
+- get the hvac systems serving the zone by using the get_list_hvac_systems_associated_with_zone function: `list_hvac_systems = get_list_hvac_systems_associated_with_zone(RMD,zone)`
+	- loop through each heating system: `for system in list_hvac_systems:`
```

Comment:
I beleive here you mean loop through each HeatingVentilatingAirconditionsSystem? 
I think for the exception should check that zone (1) has a heating system and (2) does not have cooling system. The function now only checks for (2).

Reply:
yes.  updated